modules/pipeline/providers/dbgc/dbgc.go

```diff
@@ -57,6 +60,9 @@ func (p *provider) PipelineDatabaseGC(ctx context.Context) {
 
 			p.doNotAnalyzedPipelineDatabaseGC(true)
 			p.doNotAnalyzedPipelineDatabaseGC(false)
+
+			p.doAnalyzedPipelineArchiveGC()
```

Comment:
ugly.
combine with L50.

Reply:
combine `doAnalyzedPipelineArchiveGC` and `doNotAnalyzedPipelineArchiveGC`